1-exercises/A-this/exercise-2.js

```diff
@@ -6,18 +6,20 @@
 
 */
 
-
 let person = {
-    name: "Maggie",
-    age: 26,
-    currentAddress: "London",
-    changeAddress: /* todo */
+  name: "Maggie",
+  age: 26,
+  currentAddress: "London",
+  changeAddress: function (newAddress) {
```

Comment:
I don't believe you need to use the keyword `function` here, but still great work!